### PR TITLE
KAA-1281: kaa node blocks startup if zookeeper is unavailable

### PR DESCRIPTION
**Pull request by nocs00**

### CODE REVIEW

**Comment by nocs00** [comment]
@kaa-leeroy retest this please

File: server/node/src/main/java/org/kaaproject/kaa/server/node/service/initialization/KaaNodeInitializationService.java [hunk 1]
```diff
@@ -100,18 +105,23 @@ public void start() {
...
+            if (getNodeConfig().isBootstrapServiceEnabled()) {
+                bootstrapInitializationService.start();
+            }
+            if (getNodeConfig().isOperationsServiceEnabled()) {
+                operationsInitializationService.start();
+            }
+
+            LOG.info("Kaa Node Server Started.");
+        } else {
+            LOG.error("Failed to connect to Zookeeper for {} minutes.", getNodeConfig().getZkWaitConnectionTime());
```
Comment (ikulikov) [inline server/node/src/main/java/org/kaaproject/kaa/server/node/service/initialization/KaaNodeInitializationService.java]: Failed to connect to Zookeeper within {} minutes. Kaa Node Server will be stopped.

**Comment by nocs00** [comment]
@kaa-leeroy retest this please

**Comment by nocs00** [comment]
@kaa-leeroy retest this please

**Comment by nocs00** [comment]
@kaa-leeroy retest this please

**Comment by nocs00** [comment]
@kaa-leeroy retest this please

**Pull request merged**